docs/advanced/document-store/redis.md

```diff
@@ -0,0 +1,339 @@
+(redis)=
+# Redis
+
+One can use [Redis](https://redis.io) as the document store for DocumentArray. It is useful when one wants to have faster Document retrieval on embeddings, i.e. `.match()`, `.find()`.
+
+````{tip}
+This feature requires `redis`. You can install it via `pip install "docarray[redis]".`
+````
+
+## Usage
+
+### Start Redis service
+
+To use Redis as the storage backend, it is required to have the Redis service started. Create `docker-compose.yml` as follows:
+
+```yaml
+version: "3.3"
+services:
+  redis:
+    image: redislabs/redisearch:2.6.0
```

Comment:
why it's different from the other docker image in the tests ?

Reply:
redis-stack includes something we won't need here, like graph data models
and the size redissearch docker image is half of redis-stack